README.md

```diff
@@ -1,13 +0,0 @@
-Copy Artifact Plugin
```

Comment:
I am not sure removal of readme was planned. Did you mean to replace it by the Wiki content?

Reply:
Yes, I removed the old README.md.

* It just points to Wiki and this request imports Wiki content to README.adoc.
* The note for Java < 8 is no longer valid as copyartifact requires java8.

I'm afraid it would be confusing if there were both README.adoc and README.md, and plugins.jenkins.io might work wrong.
Should I keep README.md?

Reply:
Yes, I removed the old README.md.

* It just points to Wiki and this request imports Wiki content to README.adoc.
* The note for Java < 8 is no longer valid as copyartifact requires java8.

I'm afraid it would be confusing if there were both README.adoc and README.md, and plugins.jenkins.io might work wrong.
Should I keep README.md?

Reply:
No need, sorry for confusion